server/views/partials/new_workAndSkillsPage/goals/index.njk

```diff
@@ -0,0 +1,12 @@
+{# Summary card for Goals, pulling in the relevant include #}
+
+{% from "../../../macros/summaryCardMacro.njk" import summaryCard %}
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{%- call summaryCard({title: "Goals", id: "goals"}) -%}
+
+    {% include './vc2GoalsOnly.njk' %}
+    {# {% include './plpGoalsOnly.njk' %} #}
+    {# {% include './plpAndVc2Goals.njk' %} #}
+    {# {% include './noGoals.njk' %} #}
+
```

Comment:
Plus we'll also need the include for the error content (which I think should already be there?)